daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageMajorVersion.scala

```diff
@@ -35,7 +31,9 @@ sealed abstract class LanguageMajorVersion(
 object LanguageMajorVersion {
 
   case object V1
-      extends LanguageMajorVersion(pretty = "1", stableAscending = NonEmptyList("6", "7", "8"))
+      extends LanguageMajorVersion(
+        pretty = "1",
+        stableAscending = NonEmptyList("6", "7", "8", "dev"))
```

Comment:
We should probably rename this field now since "stable" seems inappropriate for "dev". I would suggest something like
```suggestion
        minorsAscending = NonEmptyList("6", "7", "8", "dev"))
```

Reply:
good catch.